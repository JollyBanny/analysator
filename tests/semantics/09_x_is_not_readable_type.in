var a: integer;
var b: double;
var c: char;
var d: string;
var e: boolean;

begin
    read(a);
    read(b);
    read(c);
    read(d);
	read(e);
end.