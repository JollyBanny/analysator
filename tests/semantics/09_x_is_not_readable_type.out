[Line: 12; Column: 2]	boolean is not readable type
